usecases/schema/startup_cluster_sync.go

```diff
@@ -192,18 +192,23 @@ func (m *Manager) validateSchemaCorruption(ctx context.Context) error {
 	}
 	var diff []string
 	cmp := func() error {
-		if err := Equal(&m.schemaCache.State, pl.Schema); err != nil {
```

Comment:
Consider making 'err' a local variable within the 'cmp' function for improved readability and scope control.

Reply:
Ok, sure 👍